Extension fails to load

Hi, 

I just installed the linuxserver/code-server docker container and installed your extension, however it didn't work for me. The only hint what I could find is the below snippet from exthost.log. I have no idea how to continue

```
[2020-05-11 23:28:18.431] [exthost] [info] ExtensionService#_doActivateExtension keesschollaart.vscode-home-assistant {"startup":true,"extensionId":{"value":"keesschollaart.vscode-home-assistant","_lower":"keesschollaart.vscode-home-assistant"},"activationEvent":"workspaceContains:**/configuration.yaml,**/ui-lovelace.yaml"}
[2020-05-11 23:28:18.431] [exthost] [info] ExtensionService#loadCommonJSModule file:///config/extensions/keesschollaart.vscode-home-assistant-1.6.5/out/extension
[2020-05-11 23:28:18.432] [exthost] [error] Activating extension keesschollaart.vscode-home-assistant failed due to an error:
[2020-05-11 23:28:18.433] [exthost] [error] Error: Cannot find module '/config/extensions/keesschollaart.vscode-home-assistant-1.6.5/out/extension'
Require stack:
- /app/code-server/lib/vscode/out/vs/loader.js
- /app/code-server/lib/vscode/out/bootstrap-amd.js
- /app/code-server/lib/vscode/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
	at Function.Module._load (internal/modules/cjs/loader.js:840:27)
	at Function.t._load (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:920:599)
	at Function.i._load (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:889:759)
	at Function.n._load (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:886:63)
	at Module.require (internal/modules/cjs/loader.js:1019:19)
	at require (internal/modules/cjs/helpers.js:77:18)
	at Function.s [as __$__nodeRequire] (/app/code-server/lib/vscode/out/vs/loader.js:32:963)
	at p._loadCommonJSModule (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:890:966)
	at p._doActivateExtension (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:788:856)
	at p._activateExtension (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:787:937)
	at Object.actualActivateExtension (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:785:883)
	at e._activateExtension (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:566:859)
	at /app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:566:537
	at Array.map (<anonymous>)
	at e._activateExtensions (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:566:525)
	at e.activateById (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:564:310)
	at p._activateById (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:786:681)
	at p._activateIfGlobPatterns (/app/code-server/lib/vscode/out/vs/workbench/services/extensions/node/extensionHostProcess.js:792:456)
	at processTicksAndRejections (internal/process/task_queues.js:97:5)
	at async Promise.all (index 1)
	at async Promise.all (index 79)
[2020-05-11 23:28:18.433] [exthost] [info] eager extensions activated
```

Find my Home Assistant Configuration here:


Extension runs in:
- [ ] VS Code for Desktop 
- [ ] Hassio Add-on 
- [ ] Visual Studio Online
- [x] Other: Linuxserver/code-server docker image

I'm running VS Code on:
- [ ] Windows 
- [ ] Mac 
- [x] Linux 

I'm accessing my files:
- [ ] From local disk 
- [ ] Via remote SSH 
- [ ] Smb
- [x] Other:  Via mounted volume in docker container from host.

Edit: added my docker-compose.yaml
```YAML
version: "2.1"
services:
  code-server:
    image: linuxserver/code-server
    container_name: code-server
    environment:
      - PUID=984
      - PGID=984
      - TZ=Europe/Amsterdam
      - PASSWORD=${PASSWORD}
      - SUDO_PASSWORD=${SUDO_PASSWORD}
    volumes:
      - /opt/codeserver:/config
      - /opt/hassio/homeassistant:/config/workspace
    ports:
      - 8543:8443
    restart: unless-stopped
```

--- a/package.json
+++ b/package.json
@@ -246,7 +246,7 @@
   "dependencies": {
     "axios": "^0.19.2",
     "bufferutil": "^4.0.1",
-    "home-assistant-js-websocket": "5.4.1",
+    "home-assistant-js-websocket": "^5.5.0",
     "utf-8-validate": "^5.0.2",
     "vscode-extension-telemetry": "0.1.6",
     "vscode-json-languageservice": "3.7.0",

--- a/src/language-service/package.json
+++ b/src/language-service/package.json
@@ -66,7 +66,7 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "home-assistant-js-websocket": "5.4.1",
+    "home-assistant-js-websocket": "^5.5.0",
     "vscode-json-languageservice": "^3.7.0",
     "vscode-languageserver-protocol": "3.15.3",
     "vscode-uri": "2.1.2",

